### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by prestosole**
Template provided by https://github.com/auth0/open-source-template

### CODE REVIEW

**Comment by triallax** [comment]
Just curious; where did you take this from? It doesn't sound like you wrote it yourself.

**Comment by prestosole** [comment]
Seriously dude? Lol

**Comment by prestosole** [comment]
Add `Attribution` header - check

**Comment by triallax** [comment]
Just one last thing, can you make "Issues"/"Issue" lowercase?

**Comment by prestosole** [comment]
Line?

**Comment by triallax** [comment]
Just search for "Issue", there is more than a single usage.

**Comment by prestosole** [comment]
Resolved

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+2. Clone the project to your machine.
+3. Create a branch locally with a succinct but descriptive name.
+4. Commit changes to the branch.
+5. Following any formatting and testing guidelines specific to this repository.
+6. Push changes to your fork.
+7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
+
+## Getting Help
+
+Join #lacerte on Libera Chat and post your question there.
```
Comment (triallax) [inline CONTRIBUTING.md]: I would comment out this section until we create the libera channel.

**Review by triallax — CHANGES_REQUESTED** [review]

**Review by triallax — APPROVED** [review]
To be honest, I feel like the text is not suited for such a small project as Clima, so I think I may revise it later, but generally, looks good.

**Pull request merged**